proj/src/main/webapp/script.js

```diff
@@ -64,3 +64,7 @@ function SidebarCollapse () {
     // Collapse/Expand icon
     $('#collapse-icon').toggleClass('fa-angle-double-left fa-angle-double-right');
 }
+
+    // scroll down chat history to show recent message
+    var chatHistory = document.getElementById("message-list");
+    chatHistory.scrollTop = chatHistory.scrollHeight;
```

Comment:
Why is this over here by itself outside of any function?

Reply:
just deleted it; it was the change that got accidentally left behind when merging with the new chat.js